src/main/webapp/WEB-INF/pages/cart.jsp

```diff
@@ -0,0 +1,83 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+
+<jsp:useBean id="cart" type="com.es.phoneshop.model.cart.Cart" scope="request"/>
+<tags:master pageTitle="Product List">
+    <form method="post" action="${pageContext.servletContext.contextPath}/cart">
+        <p>
+            Cart: ${cart}
+        </p>
+        <p>
+            Total cost :
+            <fmt:formatNumber value="${cart.totalCost}" type="currency"
+                              currencySymbol="${cart.items[0].product.currency.symbol}"/>
+        </p>
+        <p>
+            Total quantity : ${cart.totalQuantity}
+        </p>
+        <c:if test="${not empty param.message}">
+            <div class="success" style="color:green;font-style: italic">
+                    ${param.message}
+            </div>
+        </c:if>
+        <c:if test="${not empty errors}">
+            <div class="error" style="color:red;font-weight: bold">
+                    Some of the items were not updated.
+                    See below for more info.
+            </div>
+        </c:if>
+        <table>
+        <thead>
+        <tr>
+            <td>Image</td>
+            <td>
+                Description
+            </td>
+            <td class="price">
+                Price
+            </td>
+            <td>
+                Amount
+            </td>
```

Comment:
Please format the HTML code:
```
<table>
    <thead>
        <tr>
            <td>Image</td>
            <td>Description</td>
            <td class="price">Price</td>
            <td>Amount</td>

```

More info: https://www.w3schools.com/html/html5_syntax.asp